packages/components/src/Loading/LoadingParagraph/_docs/LoadingParagraph.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Text } from "~components/Text"
+import { LoadingParagraph } from "../index"
+
+const meta = {
+  title: "Components/Loading states/LoadingParagraph",
+  component: LoadingParagraph,
+} satisfies Meta<typeof LoadingParagraph>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+}
+
+export const Animated: Story = {
+  args: { isAnimated: true },
+}
+
+export const Reversed: Story = {
+  args: { isReversed: true },
+}
+
+export const Link: Story = {
+  args: { isLink: true },
+}
+
+export const MultiLineExample: Story = {
+  render: () => (
+    <>
+      <div>
+        <LoadingParagraph />
+        <LoadingParagraph />
+      </div>
+      <Text variant="body">
+        Kaizen is Culture Amp’s design system. It’s the single source of truth
+        for our UX guidelines, design assets, and front-end code to help Culture
+        Amp’s teams rapidly create a world-class experience.
+      </Text>
+    </>
+  ),
+}
+
+export const MultiColumnExample: Story = {
+  render: () => (
+    <>
+      <div>
+        <LoadingParagraph isInline width={30} />
+        <LoadingParagraph isInline width={30} />
+        <LoadingParagraph isInline width={30} />
+      </div>
+      <div style={{ display: "grid", gridTemplateColumns: "1fr 1fr 1fr" }}>
```

Comment:
No Tailwind? 😏

Reply:
Not from the copy/paste :P I can change it, but I also have no idea what the equivalent is hahaha.

Reply:
TW grid can get a little gnarly, but for that I think it might be:
```
grid grid-cols-[repeat(3,_1fr)]
```

Reply:
That...honestly doesn't look any better 😂 More confusing if anything haha! Good to know though xD